src/function/algebra/simplify/simplifyConstant.js

```diff
@@ -150,6 +180,85 @@ export const createSimplifyConstant = /* #__PURE__ */ factory(name, dependencies
     return new OperatorNode('/', 'divide', [n, new ConstantNode(f.d)])
   }
 
+  /* Handles constant indexing of ArrayNodes, matrices, and ObjectNodes */
+  function _foldAccessor (obj, index, options) {
+    if (!isIndexNode(index)) { // don't know what to do with that...
```

Comment:
@gwhitney I see your comment, does this case still need attention, or is the current behavior ok?

Reply:
Well, I don't even know what the semantics are if the index member isn't an IndexNode object. (I don't even know if it could possibly occur that the index is not an IndexNode object.) So given that, there's no possible simplification I could think of performing. That was supposed to be the import of the comment. So let me know whether to leave the comment, delete it, or replace it with `// is this even possible?`

Reply:
I think your implementation is correct. When using an expression parsed with the expression parser I don't think it can occur that index is not an IndexNode. Maybe when someone has manually created all the nodes and simplifies that it could accidentally occur.

Thinking about it, I think you can just leave the comment as is: when the index is not an index node, it makes sense that `simplify` cannot know what to do with it, and best is to just leave it as is.

Reply:
OK, then is there anything else I need to do for this PR so that you can merge it? Thanks for creating mathjs, by the way, it's a big help to our OEIS visualization project.

Reply:
no all is ready to merge, I'll click the button 👍

Reply:
> Thanks for creating mathjs, by the way, it's a big help to our OEIS visualization project.

Thanks, good to hear 😄